hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -141,7 +141,7 @@ enum InitializingStatus {
   public OzoneContainer(
       DatanodeDetails datanodeDetails, ConfigurationSource conf,
       StateContext context, CertificateClient certClient,
-      SecretKeyVerifierClient secretKeyClient) throws IOException {
+      SecretKeyClient secretKeyClient) throws IOException {
```

Comment:
`SecretKeyClient` is the one that is actually being passed from `DataNodeStateMachine`. But I made this change to add the DNClient. So this can be reverted.